ssm/agents.py

```diff
@@ -104,35 +102,15 @@ def get_ssm_args(protocol, cp, log):
         project = None
         token = ''
 
-        use_ssl = cp.getboolean('broker', 'use_ssl')
-        if use_ssl:
-            service = STOMP_SSL_SERVICE
-        else:
-            service = STOMP_SERVICE
-
         # If we can't get a broker to connect to, we have to give up.
```

Comment:
This line seems redundant now as we're not relying on a BDII query to get brokers. Either the host's in the config or it's not.

Reply:
Removed it in next push